src/control/functor.lean

```diff
@@ -2,12 +2,28 @@
 Copyright (c) 2017 Simon Hudon. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Author: Simon Hudon
-
-Standard identity and composition functors
 -/
 import tactic.ext
 import tactic.lint
 
+/-!
+# Functors
+
+This module provides additional functor lemmas, definitions, and instances.
```

Comment:
```suggestion
This module provides additional lemmas, definitions, and instances for `functor`s.
```
so that there is an obvious link to the docs for `functor` here.